CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [v0.17.0](https://github.com/spinkube/containerd-shim-spin/releases/tag/v0.17.0) - 2024-11-08
 
```

Comment:
Can we remove `Fixed https://github.com/spinkube/containerd-shim-spin/issues/216: Spin state dir not being created`. That was not an issue in previous releases rather an issue created and resolved within this release due to moving to Spin factors / 3.0